Tests/SonarQube.TeamBuild.PreProcessor.Tests/PreProcessorTests.cs

```diff
@@ -108,6 +124,12 @@ public void PreProc_EndToEnd_SuccessCase()
             AssertExpectedLocalSetting(SonarProperties.HostUrl, "http://host", actualConfig);
             AssertExpectedLocalSetting("cmd.line1", "cmdline.value.1", actualConfig);
             AssertExpectedServerSetting("server.key", "server value 1", actualConfig);
+
+            string fxCopContent = AssertFileExists(settings.SonarConfigDirectory, TeamBuildPreProcessor.FxCopCSharpRuleset);
+            Assert.IsTrue(fxCopContent.Contains("cs.rule1"), "Ruleset does not contain the expected rule");
```

Comment:
I've replaced this assertion and the one in line 132 with calls to "AssertRuleSetContainsRules" (which I've moved to a new "PreProcessAsserts" class).